cfgov/unprocessed/css/molecules/submit-complaint-cta.less

```diff
@@ -0,0 +1,46 @@
+/* topdoc
+  name: Submit a Complaint Call to Action (CTA)
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="m-submit-complaint-cta">
+            <a href="#">Submit a Complaint</a>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-submit-complaint-cta
+            a
+  tags:
+    - cfgov-molecules
+*/
+
+.m-submit-complaint-cta {
+    float: right;
+    border-left: 1px solid @gray-50;
+    padding-left: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
+    padding-top: unit( @grid_gutter-width / 3 / @base-font-size-px, em );
+    padding-bottom: unit( @grid_gutter-width / 3 / @base-font-size-px, em );
+
+    a {
+        .webfont-medium();
+        color: @pacific;
+    }
+
+    a:before {
```

Comment:
Updated in Submit a Complaint CTA PR https://github.com/cfpb/cfgov-refresh/commit/a5b2cc149dca43fa44e78ca9c5ffe101445a6dc1